app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimAmountPage.scala.html

```diff
@@ -43,7 +44,7 @@
         errorList = form.errors.map { err =>
           ErrorLink(
             href = Some(s"#${err.key}"),
-            content = Text(s"${messages("add-claim-amount." + err.message)}"),
+            content = Text(s"${messages(s"$key."+err.message)}"),
```

Comment:
Subsitiution isn't quite right. Do you mean something like...
`s"$key.${err.message}"`